### PR TITLE
xplat: back short interval GetSystemTime calls via cpu-ticks

### PR DESCRIPTION
**Pull request by obastemur**

### CODE REVIEW

File: lib/Runtime/PlatformAgnostic/Platform/Common/HiResTimer.cpp [hunk 1]
```diff
@@ -56,18 +52,34 @@ namespace DateTime
...
     }
 
+    double HiResTimer::GetSystemTime()
+    {
+        ULONGLONG current = GetTickCount64();
+        ULONGLONG diff = current - data.cacheTick;
+
+        if (diff > 5) // max 5ms to respond system time changes
+        {
+            data.cacheSysTime = GetSystemTimeREAL();
```
Comment (obastemur) [inline lib/Runtime/PlatformAgnostic/Platform/Common/HiResTimer.cpp]: self-note: add `Assert(GetSystemTimeREAL() >= data.cacheSysTime + 5)` and add a slow test case

**Comment by jianchun** [comment]
:shipit:

**Comment by obastemur** [comment]
@curtisman @jianchun Thanks for the review

**Pull request merged**